internal/pkg/deploy/cloudformation/task.go

```diff
@@ -48,3 +54,51 @@ func (cf CloudFormation) DeployTask(input *deploy.CreateTaskResourcesInput, opts
 
 	return nil
 }
+
+// GetTasks returns all the CF stacks which represent one-off copilot tasks in a given application's environments
+func (cf CloudFormation) GetTasks(appName string) ([]deploy.TaskStackInfo, error) {
+	tasks, err := cf.cfnClient.ListStacksWithPrefix("task-")
+	if err != nil {
+		return nil, err
+	}
+	var outputTaskStacks []deploy.TaskStackInfo
+	for _, task := range tasks {
+		taskDescription, err := cf.cfnClient.Describe(aws.StringValue(task.StackName))
+		if err != nil {
+			return nil, fmt.Errorf("describe stack %s: %w", aws.StringValue(task.StackName), err)
+		}
+		var hasCopilotTag bool
+		for _, tag := range taskDescription.Tags {
+			if aws.StringValue(tag.Key) == deploy.TaskTagKey {
+				hasCopilotTag = true
+			}
+		}
+		if !hasCopilotTag {
+			continue
+		}
+		// Check the RoleArn of the task to see if it's created using a copilot EnvManagerRole
+		match := cfnExecutionRoleARNRegexp.FindStringSubmatch(aws.StringValue(taskDescription.RoleARN))
+		if len(match) == 0 {
+			continue
+		}
+		if appName == match[1] {
+			outputTaskStacks = append(outputTaskStacks, transformTask(taskDescription, appName, match[2]))
+		}
+	}
+	return outputTaskStacks, nil
+}
+
+// DeleteTask deletes a Copilot-created one-off task stack using the RoleARN that stack was created with.
+func (cf CloudFormation) DeleteTask(task deploy.TaskStackInfo) error {
+	return cf.cfnClient.DeleteAndWaitWithRoleARN(task.StackName, task.RoleARN)
+}
+
+func transformTask(in *cloudformation.StackDescription, app, env string) deploy.TaskStackInfo {
```

Comment:
Can we ditch this function since we are not reusing it and the function does not contain any complex logic?